Dockerfile

```diff
@@ -169,7 +172,11 @@ RUN mkdir DAGMC && \
 
 # Clone and install OpenMC with DAGMC
 # TODO clone a specific release when the next release containing (PR 1825) is avaialble.
-RUN git clone --shallow-submodules --recurse-submodules --single-branch --branch develop --depth 1 https://github.com/openmc-dev/openmc.git  /opt/openmc && \
+RUN cd /opt && \
+    git clone --single-branch --branch develop https://github.com/openmc-dev/openmc.git && \

```

Comment:
It seems like this line is messing with the workflow. the git instruction is not recognised.

See here https://github.com/fusion-energy/fusion_neutronics_workflow/runs/4215031584?check_suite_focus=true

Reply:
fixed that one in previous commit